adapters/richaudience/richaudiencetest/exemplary/single-banner-app.json

```diff
@@ -0,0 +1,130 @@
+{
+    "mockBidRequest": {
+      "id": "4d3f84eb-787b-42fb-a0cf-062690dadce3",
+      "test": 0,
+      "imp": [
+        {
+          "id": "div-gpt-ad-1460505748561-0",
+          "banner": {
+              "format" : [{
+                    "w": 300,
+                    "h": 250
+                  }]
+          },
+          "bidfloor": 1e-05,
+          "bidfloorcur": "USD",
+          "ext": {
+           "bidder": {
+              "pid": "OsNsyeF68q",
+              "supplyType": "site",
+              "test": true
+            }
+          }
+        }
+      ],
+      "site": {
+        "domain": "bridge.richmediastudio.com",
+        "page": "https://bridge.richmediastudio.com//ab083674fb8200b877a6983126e4477d/PrebidServer/indexRa.html?pbjs_debug=true"
+      },
```

Comment:
The test file name indicates this is if for "banner app", but there is a site section here instead.